src/components/Breadcrumb/Breadcrumb.tsx

```
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+import IconButton from '@material-ui/core/IconButton';
+import {
+  StyleRulesCallback,
+  withStyles,
+  WithStyles,
+} from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import KeyboardArrowLeft from '@material-ui/icons/KeyboardArrowLeft';
+
+import EditableText from 'src/components/EditableText';
+
+type ClassNames = 'root' | 'backButton' | 'linkText' | 'staticText';
+
+const styles: StyleRulesCallback<ClassNames> = (theme) => ({
+  root: {},
+  backButton: {
+    margin: '5px 0 0 0',
+    width: 'auto',
+    '& svg': {
+      width: 26,
+      height: 26,
+    },
+  },
+  linkText: {
+    display: 'flex',
+    alignItems: 'center',
+    color: '#3683DC',
+    textDecoration: 'underline',
+    borderColor: theme.color.grey,
+    '&:after': {
+      content: "''",
+      display: 'inline-block',
+      padding: '0 8px 0 6px',
+      height: '38px',
+      borderRight: `1px solid ${theme.color.grey1}`,
+    },
+    [theme.breakpoints.down('sm')]: {
+      display: 'none',
+    },
+  },
+  staticText: {
+    padding: '5px 10px'
+  }
+});
+
+export interface Props {
+  linkTo: string;
+  linkText: string;
+  label: string;
+  labelLink?: string | undefined;
+  errorText?: string;
+  onCancel?: () => void;
+  onEdit?: (value: string) => void;
```

Are `onCancel` and `onEdit` and `labelLink` exclusive of each other? If so, we should probably throw an error if the developer tries to use all three props and document

---

No, you can provide all three (as is the case with Linodes on any tab except "Summary"). The result is that the text is editable (by clicking on the Pencil Icon) and also clickable (it tabs back to Summary).